### PR TITLE
Slight changes to match production

### PR DESCRIPTION
**Pull request by MTCam**
Match mostly cosmetic differences between main and production, adds an esdg entropy-to-conserved 
conversion util test to `test_euler.py`.

This change set trivially reduces the difference between production and main, but helps eliminate some merge conflicts in current developments (mostly Tulio's) when running production tests. 

**Questions for the review**:
- [ ] Is the scope and purpose of the PR clear?
  - [ ] The PR should have a description.
  - [ ] The PR should have a guide if needed (e.g., an ordering).
- [ ] Is every top-level method and class documented? Are things that should be documented actually so?
- [ ] Is the interface understandable? (I.e. can someone figure out what stuff does?) Is it well-defined?
- [ ] Does the implementation do what the docstring claims?
- [ ] Is everything that is implemented covered by tests?
- [ ] Do you see any immediate risks or performance disadvantages with the design? Example: what do interface normals attach to?

### CODE REVIEW

File: test/test_euler.py [hunk 1]
```diff
@@ -225,6 +225,176 @@ def inf_norm(x):
...
 
 
+@pytest.mark.parametrize("nspecies", [0, 10])
+@pytest.mark.parametrize("dim", [1, 2, 3])
+@pytest.mark.parametrize("order", [2, 3, 4])
+def test_entropy_to_conserved_conversion(actx_factory, nspecies, dim, order):
+    """Test the inviscid rhs using a trivial constant/uniform state.
+
+    This state should yield rhs = 0 to FP.  The test is performed for 1, 2,
+    and 3 dimensions, with orders 1, 2, and 3, with and without passive species.
```
Comment (tulioricci) [inline test/test_euler.py:235]: 2, 3, 4*

File: test/test_viscous.py [hunk 2]
```diff
@@ -401,6 +401,8 @@ def test_diffusive_heat_flux(actx_factory):
     # assemble d_alpha so that every species has a unique j
     d_alpha = np.array([(ispec+1) for ispec in range(nspecies)])
 
+    # h_alpha = np.array([1./(ispec+1) for ispec in range(nspecies)])
```
Comment (tulioricci) [inline test/test_viscous.py:404]: Needed?

File: test/test_euler.py (hunk 1 above)
Comment (MTCam) [reply test/test_euler.py:235]: Hrm, no?  This comment seems consistent with what is tested in the test where it appears.

**Review by tulioricci — APPROVED** [review]
:+1:

**Pull request merged**